src/js/components/search/filters/timePeriod/TimePeriod.jsx

```diff
@@ -310,7 +311,7 @@ export default class TimePeriod extends React.Component {
                                 aria-label="Fiscal Year"
                                 title="Fiscal Year"
                                 onClick={this.toggleFilters}>
-                                Fiscal Year
+                                Fiscal Year<GlossaryLink term="fiscal-year-fy" />
```

Comment:
do you need to style this glossary icon so that it's gray not blue?

Reply:
good catch.
fixed.